### DISCUSSION TITLE
登录问题

### DISCUSSION THREAD

**Discussion by MUYI-XIAN** [simple-robot/Discussions#101] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

code=6, title=登录失败, message=服务连接中，请稍后再试。(0x6), errorInfo=)
请问这是什么原因，我看了配置文档，好像更新了，按照新的配置文档进行重写也没办法解决

**Comment by ForteScarlet** [comment]
看上去像是什么网络问题，或许可以去[mirai的issues](https://github.com/mamoe/mirai/issues)中搜搜相关信息，或者换个号试试之类的
